src/printer.js

```diff
@@ -2067,6 +2067,16 @@ function genericPrintNoParens(path, options, print, args) {
         join(", ", path.map(print, "parameters")),
         ")"
       ])
+    case "TSNamespaceExportDeclaration":
+      parts.push(
+        "export as namespace ", 
```

Comment:
Huh. I've been using TS for years and never knew the `export as namespace` feature.